### PR TITLE
Add CS 361 as a separate option

### LINKED ISSUE #206: Add support for 361
Some 361 required C libraries are not included in the image. I don't have the specific libraries yet, will be testing later this week. Will mark down here what is missing specifically.

### PR DESCRIPTION
**Pull request by Advill**
Closes #206

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,28 @@ apt will require more extensive modifications.
     * Reference solution manual page (`y86ref(1)`)
     * Simulator (`ysim`)
 
+### CS361
```
[the remark this answers is not in the document]
Comment (Advill) [reply README.md]: Ah yes, completely forgot about that this time around, thanks for the reminder.

File: roles/task-shortcuts/files/uug_ansible_wrapper.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
 # checkbox
 # Map of course names to the Ansible tags
 COURSES = {'CS 101': 'cs101', 'CS 149': 'cs149', 'CS 159': 'cs159',
-           'CS 261': 'cs261', 'CS 354': 'cs354'}
+			'CS 261': 'cs261', 'CS 361': 'cs361',  'CS 354': 'cs354'}
```
Comment (kylelaker) [inline roles/task-shortcuts/files/uug_ansible_wrapper.py]: Can you fix the white space here? We want the first item on the second line to be aligned with the first item on the first line.

```suggestion
           'CS 261': 'cs261', 'CS 361': 'cs361',  'CS 354': 'cs354'}
```

Comment (Advill) [reply roles/task-shortcuts/files/uug_ansible_wrapper.py]: Sure thing, I think vim had a fit with that.

**Comment by ripleymj** [comment]
If this is the consensus on how to proceed with 361, I'm ok with it. Let me know if I need to cast a tie-breaker vote on anything.

**Comment by kylelaker** [comment]
@ripleymj It's probably the right way to prevent future issues with the script, etc. I also think it makes expanding 361 easier in the future. If after a few semesters they haven't been split more then we can always merge them.

File: roles/task-shortcuts/files/uug_ansible_wrapper.py [hunk 3]
```diff
@@ -28,7 +28,7 @@
 # checkbox
 # Map of course names to the Ansible tags
 COURSES = {'CS 101': 'cs101', 'CS 149': 'cs149', 'CS 159': 'cs159',
-           'CS 261': 'cs261', 'CS 354': 'cs354'}
+           'CS 261': 'cs261', 'CS 361': 'cs361',  'CS 354': 'cs354'}
```
Comment (showalrm) [inline roles/task-shortcuts/files/uug_ansible_wrapper.py]: Should 361 and 354 be switched here as well?

Comment (kylelaker) [inline roles/task-shortcuts/files/uug_ansible_wrapper.py]:
```suggestion
           'CS 261': 'cs261', 'CS 354': 'cs354', 'CS 361': 'cs361'}
```

**Comment by kylelaker** [comment]
I am going to proceed with merging this. As we need more things for 361, that can be tracked in separate issues and pull requests.

Thanks for your help with this, @Advill! Next time we'll get better and choosing a direction before you start work :)

**Pull request merged**